xwiki-platform-core/xwiki-platform-refactoring/xwiki-platform-refactoring-default/src/main/java/org/xwiki/refactoring/internal/DefaultModelBridge.java

```diff
@@ -179,20 +179,33 @@ public boolean copy(DocumentReference source, DocumentReference destination)
 
     @Override
     public boolean delete(DocumentReference reference)
+    {
+        return delete(reference, true);
+    }
+
+    @Override
+    public boolean expurge(DocumentReference reference)
+    {
+        return delete(reference, false);
+    }
+
+    private boolean delete(DocumentReference reference, boolean totrash)
```

Comment:
haha, see you need this method actually ;) I feel it's much more explicit than trying to understand what delete() and expurge() do.

Reply:
BTW it's not trash, it's recycle bin, you need consistency. BTW I didn't check if we were using RecycleBin or Trash elsewhere but better be consistent.

Reply:
Naming convention: `totrash` --> `toTrash``

PS: yes I know it's not been followed in the past... unfortunately.

Reply:
I'm using `toRecycleBin` for the new code. I propose to migrate the `toTrash` to `toRecyclebin` gradually.